core/src/main/java/io/cucumber/core/options/PluginOption.java

```diff
@@ -44,16 +42,13 @@
 
     static {
         Map<String, Class<? extends Plugin>> plugins = new HashMap<>();
-        plugins.put("default_summary", DefaultSummaryPrinter.class);
         plugins.put("html", HtmlFormatter.class);
         plugins.put("json", JsonFormatter.class);
         plugins.put("junit", JUnitFormatter.class);
-        plugins.put("null_summary", NullSummaryPrinter.class);
         plugins.put("pretty", PrettyFormatter.class);
         plugins.put("progress", ProgressFormatter.class);
         plugins.put("message", MessageFormatter.class);
         plugins.put("rerun", RerunFormatter.class);
-        plugins.put("summary", DefaultSummaryPrinter.class);
```

Comment:
>  This solution assumes that there is no need to be able to add the DefaultSummaryPrinter (without using full qualified name) from the JUnit and TestNG runners.

I made that same assumption in the past but people have requested a summary along with a summary of their JUnit tests. 

I don't think `--plugin summary` will be problematic though. Plugins are stored in a set. This avoids duplicate plugins.  I also don't think `--plugin summary --no-summary` is a problem either. If a bit redundant.

Reply:
`--plugin summary` added back in https://github.com/cucumber/cucumber-jvm/pull/2197/commits/d0ba0bbe774b2b6eee7c44156d4f5827806b4fdd